src/Confluent.Kafka.Avro/Confluent.Kafka.Avro.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net452;</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Confluent.Apache.Avro" Version="1.0" />
```

Comment:
Dont forget to update this version to reflect the avro version

Reply:
done